src/Traits/FastRefreshDatabase.php

```diff
@@ -48,16 +48,23 @@ protected function refreshTestDatabase()
      */
     protected function calculateMigrationChecksum(): string
     {
-        $finder = Finder::create()
-            ->in(database_path('migrations'))
-            ->name('*.php')
-            ->ignoreDotFiles(true)
-            ->ignoreVCS(true)
-            ->files();
-
-        $migrations = array_map(static function (SplFileInfo $fileInfo) {
-            return [$fileInfo->getMTime(), $fileInfo->getPath()];
-        }, iterator_to_array($finder));
+        // Get all loaded migration paths.
+        $migrations = collect(app('migrator')->paths())
+            // There might be loaded paths that does not exist. We need to filter those out.
+            ->map(fn ($path) => realpath($path))
+            ->filter()
+            ->flatMap(function ($path) {
+                $finder = Finder::create()
+                    ->in($path)
+                    ->name('*.php')
+                    ->ignoreDotFiles(true)
+                    ->ignoreVCS(true)
+                    ->files();
+
+                return array_map(static function (SplFileInfo $fileInfo) {
+                    return [$fileInfo->getMTime(), $fileInfo->getPath()];
+                }, iterator_to_array($finder));
+            })->toArray();
```

Comment:
```suggestion
        $paths = [
            database_path('migrations'),
            ...app('migrator')->paths(),
        ];

        $finder = Finder::create()
            ->in($paths)
            ->name('*.php')
            ->ignoreDotFiles(true)
            ->ignoreVCS(true)
            ->files();

        $migrations = array_map(static function (SplFileInfo $fileInfo) {
            return [$fileInfo->getMTime(), $fileInfo->getPath()];
        }, iterator_to_array($finder));
```

What about this instead? Might be a little bit easier to read

Reply:
Yeah, thats better!

Reply:
Maybe we should even extract the paths into a method instead?

Right now you'd have to duplicate the full calculateMigrationChecksum method in order to customize the paths.
I'm thinking this is quite similar to how event/listener discovery is structured in Laravel itself, which is also customizable.

See: https://github.com/laravel/framework/blob/bef771d8dc0db952cc93d54824b304ab21405f97/src/Illuminate/Foundation/Support/Providers/EventServiceProvider.php#L143-L148

Reply:
Sure, why not, it makes it more customizable.

Do you want med to fix it?

Reply:
I unfortunately can't merge this either, but I think it would be a worthwhile addition.

@Sammyjo20 what do you think about this?

Reply:
Not checked with Sam but I reckon the method approach would be cleaner. 

We will then review and test it.

Reply:
I'll update this PR with your proposed solution as soon as I can.